orca/request_opendap.py

```diff
@@ -5,26 +5,25 @@
 from .split import split_url
 
 
-def request_opendap(url):
+def request_opendap(url, logger, req_limit=5e8):
     data = open_dataset(url)
     size = data.nbytes
 
-    if size / 2 > 5e8:
-        urls = split_url(url, size)
-    else:
-        urls = [url]
+    logger.info(f"Splitting url: {url}")
+    urls = split_url(url, size, req_limit) if size / 2 > req_limit else [url]
```

Comment:
Now that I think about it a bit more it may make more sense for this line to be:
```
urls = split_url(url, size)
```
The logic that we are using the decide whether or not to call the method could just be handled inside the method itself. Furthermore this allows us to pull out the `req_limit` default var from the `request_opendap` and place it into `split_url` where it is actually used.